src/Stepper/Stepper.less

```diff
@@ -0,0 +1,24 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+.Stepper {
+  .flexbox();
+}
+
+.Stepper--sidebar {
+  .flex(0, 1, 18rem);
+  max-width: 18rem;
+  .padding--l;
+}
+
+.Stepper--stepsDisplay {
+  list-style-type: none;
```

Comment:
let's switch to the `.list--reset()` mixin for consistency:
e.g. https://github.com/Clever/components/search?q=list-reset&unscoped_q=list-reset